main.tf

```diff
@@ -1,7 +1,8 @@
 locals {
-  port             = "${var.port == "" ? "${var.engine == "aurora-postgresql" ? "5432" : "3306"}" : var.port}"
-  master_password  = "${var.password == "" ? random_id.master_password.b64 : var.password}"
-  backtrack_window = "${var.backtrack_window == "" ? "${var.engine == "aurora" ? "0" : ""}" : var.backtrack_window}"
+  port                = "${var.port == "" ? "${var.engine == "aurora-postgresql" ? "5432" : "3306"}" : var.port}"
+  master_password     = "${var.password == "" ? random_id.master_password.b64 : var.password}"
+  backtrack_window    = "${var.backtrack_window == "" ? "${var.engine == "aurora" ? "0" : ""}" : var.backtrack_window}"
+  ca_cert_identifier  = "${var.ca_cert_identifier == "" ? "rds-ca-2015" : var.ca_cert_identifier}"
```

Comment:
`local.ca_cert_identifier` is not needed, because `var. ca_cert_identifier` already has the same value as default. Please update.

Reply:
thanks. Update now.